drv/spi-msg/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "drv-spi-msg"
```

Comment:
Since this is a client API, it should probably be named `drv-spi-msg-api` and be in a `spi-msg-api` folder.

Reply:
I did a rename of all spirot variants to "sprot" with drv/spi-msg changing to drv/sprot-api.